internal/cli/atlas/streams/instance/download_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instance
+
+import (
+	"io"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/test"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20231115008/admin"
+)
+
+func TestDownloadOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockStreamsDownloader(ctrl)
+
+	const contents = "expected"
+	const projectID = "download-project-id"
+	const tenantName = "streams-tenant"
+
+	file, err := os.CreateTemp("", "")
+	if err != nil {
+		require.NoError(t, err)
+	}
+	filename := file.Name()
+	defer os.Remove(filename)
```

Comment:
```suggestion
	t.Cleanup(func(t){ os.Remove(filename)})
```

Reply:
Ended up removing all of this temp file code, since we only actually care that we are returning a readerCloser (we're mocking it all anyway). 

Fixed in 488dbf4ed